srcpkgs/forgejo/template

```diff
@@ -0,0 +1,70 @@
+# Template file for 'forgejo'
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# - [Installation from source](https://forgejo.org/docs/next/contributor/from-source)
+# - [Forgejo v9.0 is GPLv3+](https://codeberg.org/forgejo/forgejo/commit/94631c)
+
+homepage="https://forgejo.org"
+
+# https://github.com/void-linux/void-packages/blob/master/Manual.md#available_vars
+
+# Forgejo accepts contributions compatible with the GPLv3-or-later license.
+license="GPL-3.0-or-later"
+
+maintainer="Markus Heiser <markus.heiser@darmarit.de>"
+pkgname=forgejo
+# https://forgejo.org/docs/next/contributor/release/#release-cycle
+version=9.0.0
+revision=1
+_version="$(echo "${version}" | tr '.' '_')"
+short_desc="Forgejo is a self-hosted lightweight software forge"
+
+# go-bindata is from commit (dated in Oct 23, 2015):
+# - https://github.com/jteeuwen/go-bindata/commit/a0ff2567cf
+# - https://github.com/void-linux/void-packages/blob/059fd3968/srcpkgs/go-bindata/template#L5
+# go-bindata/go-bindata is more updated compared to jteeuwen/go-bindata
+# - https://github.com/go-bindata/go-bindata/releases
+
+hostmakedepends="go-bindata"
+makedepends="sqlite-devel pam-devel"
+depends="git git-lfs bash"
+
+distfiles="https://codeberg.org/forgejo/forgejo/releases/download/v${version}/forgejo-src-${version}.tar.gz"
+checksum=21364d6c1635711189f25da5dc343b3b28e8ade20a5f00202301ccc364adc1d2
+changelog="https://codeberg.org/forgejo/forgejo/src/branch/forgejo/RELEASE-NOTES.md#${_version}"
+
+# - https://github.com/void-linux/void-packages/blob/master/Manual.md#build-style-scripts
+# - https://github.com/void-linux/void-packages/blob/master/Manual.md#go-packages
+build_style=go
+go_ldflags=" -X main.Version=${version}"
+
+# forgejo is still module code.gitea.io/gitea
+go_import_path="code.gitea.io/gitea"
+
+# Follow forgejo default release build and enable pam support additionally
+go_build_tags="bindata timetzdata sqlite sqlite_unlock_notify pam"
+
+# https://github.com/void-linux/void-packages/blob/master/Manual.md#system-accounts
+system_accounts="_forgejo"
+_forgejo_homedir="/var/lib/forgejo"
+_forgejo_shell="/bin/bash" # Proper shell needed for ssh support
```

Comment:
What does this mean? It will not work with `/bin/sh`?

Reply:
See below in https://github.com/void-linux/void-packages/pull/52694#discussion_r1875988312